components/popover/popover-mixin.js

```diff
@@ -362,12 +381,17 @@ export const PopoverMixin = superclass => class extends superclass {
 			</div>
 		` : nothing;
 
-		return html`${content}${pointer}`;
+		return this._mobileTrayLocation ? html`
+			${content}
+			<d2l-backdrop for-target="content-wrapper" ?shown="${this._showBackdrop}"></d2l-backdrop>
+			${pointer}
+		` : html`${content}${pointer}`;
```

Comment:
Super nit, but I find this easier to grok and it's maybe a bit more resilient to future changes to the layout since it isn't duplicated twice:

```suggestion
		const backdrop = this._mobileTrayLocation ?
			html`<d2l-backdrop for-target="content-wrapper" ?shown="${this._showBackdrop}"></d2l-backdrop>` :
			nothing;
		return html`${content}${backdrop}${pointer}`;
```